11-checkout.md

```diff
@@ -1,119 +0,0 @@
----
-layout: lesson
-subtitle: Instructor Checkout Procedure
----
-> ## Motivating Question
-> 
-> What are the final steps in becoming an instructor trainer?
-{: .question}
-
-The final steps in getting certified are as follows:
-
-1. Pick one of the lessons we are currently accepting trainee requests on
-   and submit a *small* change.
-   (Your trainer will provide you with a list of lessons.)
-   If you want to teach Software Carpentry,
-   you must submit the change as a pull request on GitHub
-   and send the link to your trainer.
-   If you want to teach Data Carpentry,
-   go to [its submission page][dc-submission-page]
-   (which will redirect you to the appropriate place).
-
-   Note:
-   you may be allowed or encouraged to submit a change to a Data Carpentry lesson
-   when qualifying as a Software Carpentry instructor---please
-   check with your trainer for the current status of this.
-
-2. Remember that lesson maintainers will be getting a number of pull requests like this,
-   so take some time to write a good commit message
-   so that the lesson maintainer understands what you're doing and why you think it's useful.
-   For example:
-
-   ~~~
-   04-changes: Add a 'commit --amend' exercise
-
-   Everybody will botch a commit message at some point.  This
-   exercise introduces them to an easy way to fix those mistakes.
-   ~~~
-
-   (as both a commit message and a pull-request summary) will be a lot
-   easier for the maintainer than:
-
-   ~~~
-   Instructor-training assignment
-   ~~~
```

Comment:
Shouldn't this title be at the top? The submission to the materials is also an assignment. ?